recipes/gaia/meta.yaml

```diff
@@ -1,8 +1,6 @@
-{% set version = "0.1.0" %}
-
 package:
   name: gaia
-  version: {{ version }}
+  version: {{ environ['GIT_DESCRIBE_TAG'] }}
```

Comment:
Ah, nice, I wasn't aware of this environment variable!  Hard to complain about this since it doesn't seem to be used anywhere else in the recipe.

+1

Reply:
I just learned about this as well. We do an uglier thing in uvcdat so we'll update that as well when we get a chance.

Reply:
Looking at the vcs package and think there we want to include the date as well which make it more complex.